exporter/datadogexporter/translate_traces.go

```diff
@@ -284,9 +284,20 @@ func spanToDatadogSpan(s ptrace.Span,
 		tags[tracetranslator.TagW3CTraceState] = string(s.TraceState())
 	}
 
-	// get events as just a general tag
-	if s.Events().Len() > 0 {
-		tags[eventsTag] = eventsToString(s.Events())
+	if eventsLen := s.Events().Len(); eventsLen > 0 {
```

Comment:
Note that we are in the process of moving part of the code to the Datadog Agent (see #9693, pending stability of the `pdata` module) so it may take a little until this is mergeable